crypto/poly1305/poly1305_ameth.c

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2007-2016 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
I assumed that the 2007 start copyright remained since this was derived from the HMAC code with large swaths being nearly identical.